openwisp_firmware_upgrader/tests/test_private_storage.py

```diff
@@ -68,9 +74,41 @@ def test_staff_operator_with_same_organization(self):
             user=staff_user, organization=self.test_org, is_admin=True
         )
         self.client.force_login(staff_user)
-        self._download_firmware_assert_status(200)
+        self._download_firmware_assert_status(403)
```

Comment:
Why did you change this? Aren't operators allowed to view the FirmwareImage? 

https://github.com/openwisp/openwisp-firmware-upgrader/blob/ee878d5f080059b0b47ca3246c4fa207c043ad43/openwisp_firmware_upgrader/migrations/__init__.py#L28-L34